nevergrad/functions/images/imagelosses.py

```diff
@@ -113,11 +117,17 @@ def koncept(self) -> tp.Any:  # cache the model
         if key not in MODELS:
             if os.name != "nt":
```

Comment:
can you disentangle all this?
if os.name == "nt" then raise...
so that we don't need an "else" clause, making it easier to follow